.github/workflows/audit.yml

```diff
@@ -0,0 +1,16 @@
+name: Security audit
+on:
+  push:
+    paths: 
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  schedule:
+    - cron: '19 20 1 * *'
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/audit-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this token already defined somewhere?

Reply:
Yes, it's part of the actions environment, used to grant write access to the task so it can post a summary of any audit failures. A similar key is used when we invoke `cargo clippy` in tests.yml.